src/main/java/calculator/Spliter.java

```diff
@@ -0,0 +1,53 @@
+package calculator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Spliter {
+
+    public static final String DELIMITER1 = "[,|:]";
+    public static final String CUSTOMER_DELIMITER = "//(.)\n(.*)";
+
+    public String[] splitValue(String value) {
+        Matcher m = Pattern.compile(CUSTOMER_DELIMITER).matcher(value);
```

Comment:
여기서 Pattern.complie 은 항상 같은 Pattern 을 가리키잖아요?
Pattern 은 생성 비용이 높기 때문에 상수로 초기화시키고 사용하는 것이 좋다고 하더라구요! 
```suggestion
        private static final Pattern CUSTOM_PATTERN = Pattern.compile(CUSTOMER_DELIMITER);
```

아, 그리고 이 클래스의 멤버변수는 private 이어도 될 것 같습니다! Splitter 클래스만 알고 있어도 되는 변수로 보여요!

Reply:
https://javabom.tistory.com/30